client/web/compose/src/components/Public/Record/BulkEdit/index.vue

```diff
@@ -179,6 +184,10 @@ export default {
     getFieldSelectorPlaceholder () {
       return this.$t(`recordList.bulkRecord.field.add${this.fields.length ? 'Another' : ''}`)
     },
+
+    selectedQuery () {
+      return this.filterQuery + this.selectedRecords.map(id => `recordID=${id}`).join(' OR ')
```

Comment:
return `this.selectedRecords.length ? this.selectedRecords.map(id => `recordID=${id}`).join(' OR ') : this.query`